app/controllers/ReviewApplicationController.scala

```diff
@@ -33,16 +34,14 @@ abstract class ReviewApplicationController(applicationClient: ApplicationClient)
     implicit user =>
       val personalDetailsFut = applicationClient.findPersonalDetails(user.user.userID, user.application.applicationId)
       val assistanceDetailsFut = applicationClient.findAssistanceDetails(user.user.userID, user.application.applicationId)
-      val frameworkLocationFut = getSelection(user.application.applicationId)
 
       (for {
         gd <- personalDetailsFut
         ad <- assistanceDetailsFut
-        fl <- frameworkLocationFut
       } yield {
-        Ok(views.html.application.review(gd, ad, fl, user.application))
+        Ok(views.html.application.review(gd, ad, LocationAndSchemeSelection.empty, user.application))
```

Comment:
So we do not have scheme preferences when we present the review page, is that right?